<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReorientDialog</class>
 <widget class="QDialog" name="ReorientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>541</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Reorient</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>321</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Setting Angles</string>
   </property>
   <widget class="QTableWidget" name="DataTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>300</width>
      <height>270</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="printButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>500</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>500</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>500</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Reorientation Type</string>
   </property>
   <widget class="QRadioButton" name="XAxisButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Single Axis</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="ZoneButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Double Axis</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>271</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Reorientation Axis</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X-Axis (beam axis)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Z-Axis (camera-axis)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="XHLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="XKLineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="XLLineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ZHLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ZKLineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ZLLineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="XGoButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>431</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Orientation Axis</string>
   </property>
   <widget class="QRadioButton" name="XAxisRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>103</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X-Axis</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="YAxisRadioButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>103</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y-Axis</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AAxisRadioButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>103</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Arbitary</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ZAxisRadioButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>103</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Z-Axis</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AXHLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="AXLLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="AZLLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="AZKLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AXKLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AZHLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="AxesLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Angle Range</string>
   </property>
   <widget class="QRadioButton" name="AngleRangePM">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-180 &lt; x &lt; 180</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="AngleRangeP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>103</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0 &lt; x &lt; 360</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>XHLineEdit</tabstop>
  <tabstop>XKLineEdit</tabstop>
  <tabstop>XLLineEdit</tabstop>
  <tabstop>ZHLineEdit</tabstop>
  <tabstop>ZKLineEdit</tabstop>
  <tabstop>ZLLineEdit</tabstop>
  <tabstop>XGoButton</tabstop>
  <tabstop>DataTable</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>AAxisRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>AXHLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AAxisRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>AXKLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AAxisRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>AXLLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AAxisRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>AZHLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AAxisRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>AZKLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AAxisRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>AZLLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
